roles/openshift_grafana/templates/grafana.yml.j2

```diff
@@ -97,7 +97,7 @@ spec:
         - mountPath: /etc/proxy/secrets
           name: grafana-proxy-secrets
         command:
-         - "./bin/grafana-server"
+         - "./bin/linux-amd64/grafana-server"
```

Comment:
This seems rather strange and very different from the upstream Grafana container image, how are we building this?

Reply:
this is how I find it working for both deb and centos.

this is how I built it, it's quite old we have a newer file
https://raw.githubusercontent.com/mrsiano/openshift-grafana/master/Dockerfile